frontend/src/components/settings/general/TimeZoneSelector.tsx

```diff
@@ -37,13 +38,13 @@ const TimeZoneSelector = ({
     <>
       <Row className="full-select d-flex flex-row align-items-center gap-2">
         <Col>
-          <Row>
-            <span className="title">{t('settings:general.timeZone')}</span>
-            <small className="description p-0">
-              {t('settings:general.timeZoneDescription')}
-            </small>
-          </Row>
+          <span className="title">{t('settings:general.timeZone')}</span>
+
+          <small className="description p-0">
```

Comment:
maybe better to also put this padding into the description class? or it there a different reason as why we put it here?